unit/util/expr_cast/expr_cast.cpp

```diff
@@ -149,6 +149,7 @@ SCENARIO("expr_dynamic_cast",
       ID_byte_extract_little_endian,
       symbol_exprt(typet()),
       constant_exprt("0", typet()),
+      8,
```

Comment:
At least `CHAR_BIT`?  Or some other macro?

Reply:
Actually it doesn't matter, I've replaced this by `1, // arbitrary integer`.